awscli/examples/medical-imaging/get-datastore.rst

```diff
@@ -9,14 +9,34 @@ The following ``get-datastore`` code example gets a data store's properties. ::
 Output::
 
     {
-    "datastoreProperties": {
-        "datastoreId": "12345678901234567890123456789012",
-        "datastoreName": "TestDatastore123",
-        "datastoreStatus": "ACTIVE",
-        "losslessStorageFormat": "JPEG_2000_LOSSLESS",
-        "datastoreArn": "arn:aws:medical-imaging:us-east-1:123456789012:datastore/12345678901234567890123456789012",
-        "createdAt": "2022-11-15T23:33:09.643000+00:00",
-        "updatedAt": "2022-11-15T23:33:09.643000+00:00"
+        "datastoreProperties": {
+            "datastoreId": "12345678901234567890123456789012",
+            "datastoreName": "TestDatastore123",
+            "datastoreStatus": "ACTIVE",
+            "losslessStorageFormat": "HTJ2K"
+            "datastoreArn": "arn:aws:medical-imaging:us-east-1:123456789012:datastore/12345678901234567890123456789012",
+            "createdAt": "2022-11-15T23:33:09.643000+00:00",
+            "updatedAt": "2022-11-15T23:33:09.643000+00:00"
+        }
+    }
+
+The following ``get-datastore`` code example gets a data store's properties with store configured for JPEG 2000 Lossless storage format. ::
```

Comment:
1. This needs a `**Example 2: ....**` title line. Please add `Example 1:` to the existing example title too. 
2. Do you feel the example is useful? It's identical CLI input, so it's not teaching a different way to use the CLI, but rather just showing how the output is different for a different resource.

Reply:
Sounds good. Still missing the titles with the `Example 1:` and `Example 2:` in the latest revision.